ruby/ql/test/library-tests/ast/TreeSitter.expected

```diff
@@ -2041,14 +2037,10 @@ control/cases.rb:
 #   57|           1: [ReservedWord] :
 #   57|         3: [ReservedWord] ,
 #   57|         4: [HashSplatNil] **nil
-#   57|           0: [ReservedWord] **
```

Comment:
This doesn't look right, `**` and `nil` are separate tokens. It is possible to write things like

```
def foo(** 
    # comment
   nil) end
```
In this case you'd still end up with overlapping `comment` and `hashsplatnil` tokens.

Reply:
I wonder why `HashSplatNil` is a `Token` in the first place then, when actually it is defined as the sequence of two tokens?